zenmap/install_scripts/macosx/launcher.sh

```diff
@@ -41,14 +41,17 @@ if [ -z ${lang+x} ]; then
   export LANG="`echo $lang`.UTF-8"
 fi
 
-echo $LANG > ~/Desktop/tmp.txt
```

Comment:
The following lines were actually a leftover that I wrote a while ago to debug the chosen language.